packages/world/script/utils/config.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
+
+import { IWorld } from "../../src/codegen/world/IWorld.sol";
+
+struct Config {
+  IWorld world;
+  uint256 fromBlock;
+  string indexerUrl;
+}
+
+function config() returns (Config memory) {
+  Config memory cfg = _getChainConfig();
+  StoreSwitch.setStoreAddress(address(cfg.world));
+  return cfg;
+}
+
+function _getChainConfig() view returns (Config memory) {
+  if (block.chainid == 690) {
```

Comment:
nice to have: in the future we could read this from `worlds.json` directly since you can read JSON in forge scripts right

Reply:
Yup